graql/reasoner/type-hierarchy.feature

```diff
@@ -509,6 +509,8 @@ Feature: Type Hierarchy Resolution
 
 
   # TODO: re-enable all steps once attribute re-attachment is resolvable
+  #  TODO: disabled because down and side casting no longer allowed
+  @ignore
```

Comment:
Same comment as above - if we're settled on banning type generation, we should delete this scenario entirely

Reply:
deleted